executor/common.h

```diff
@@ -689,9 +688,8 @@ static void loop(void)
 			if (current_time_ms() - start < program_timeout_ms)
 				continue;
 #else
-		if (current_time_ms() - start < /*{{{PROGRAM_TIMEOUT_MS}}}*/) {
```

Comment:
I'm afraid [this broke clang](https://syzkaller.appspot.com/bug?id=e73a487b4fd7623d72ee7581490a0141d7d6b5f5):
```
                
                <stdin>:309:4: error: misleading indentation; statement is not part of the previous 'if' [-Werror,-Wmisleading-indentation]
                                        kill_and_wait(pid, &status);
                                        ^
                <stdin>:307:3: note: previous statement is here
                                if (current_time_ms() - start < 5000)
                                ^
                1 error generated.
                
                compiler invocation: c++ [-o /tmp/syz-executor3267117645 -DGOOS_openbsd=1 -DGOARCH_amd64=1 -DHOSTGOOS_openbsd=1 -x c - -m64 -static -lutil -O2 -pthread -Wall -Werror -Wparentheses -Wunused-const-variable -Wframe-larger-than=16384]
```

Reply:
@mptre FYI

Reply:
This is https://bugs.llvm.org/show_bug.cgi?id=48664
Trying to fix this in #2795